sdk/keyvault/keyvault-certificates/samples/javascript/mergeCertificate.js

```diff
@@ -58,6 +65,8 @@ ${base64Csr}
   await client.mergeCertificate("MyCertificate", [Buffer.from(base64Crt)]);
 }
 
+module.exports = { main };
```

Comment:
Yes, I'd like to get rid of this and modify the prep-samples script to insert it, but I will have to do it for every sample at the same time in another PR. I _could_ make changes to the storage ones in this PR, but I have another PR for some fixes to storage samples open right now so I think it would be best to do them all mechanically in a third PR.

Reply:
Let's use a separate PR to make that change in both keyvaults and storage at the same time and use this PR to get keyvault samples to be in a runnable state by scripts